test/item_repository_test.rb

```diff
@@ -148,18 +148,75 @@ def test_find_all_by_created_at
   end
 
   def test_find_all_by_updated_at
-    output_1 = item_repository.find_all_by_updated_at(DateTime.parse('2012-03-27 14:53:59 UTC')).count
+    output_1 = item_repository.find_all_by_updated_at(Date.parse('2012-03-27 14:53:59 UTC')).count
     expected_1 = 10
-    output_2 = item_repository.find_all_by_updated_at(DateTime.parse('2015-03-27 14:53:59 UTC'))
+    output_2 = item_repository.find_all_by_updated_at(Date.parse('2015-03-27 14:53:59 UTC'))
     expected_2 = []
 
     assert_equal expected_1, output_1
     assert_equal expected_2, output_2
   end
 
+  def test_most_revenue_returns_top_x_item_instances_ranked_by_total_revenue
+    load_data
+    result_1 = @se.item_repository.most_revenue(1)[0].id
+    result_2 = @se.item_repository.most_revenue(2).count
+    assert_equal 63, result_1
+    assert_equal 2, result_2
+  end
+
+  def test_most_items_returns__top_x_item_instances_ranked_by_total_number_sold
+    load_data
+    result_1 = @se.item_repository.most_items(1)[0].id
+    result_2 = @se.item_repository.most_items(3).count
+    assert_equal 63, result_1
+    assert_equal 3, result_2
+  end
+
   private
 
   def fixtures_directory
     File.expand_path('../../data/fixtures', __FILE__)
   end
+
+  def load_data
+    repo_data = {
+      items: [
+        { id: 61, merchant_id: 10, unit_price: 10_000, created_at: "2012-03-24", updated_at: "2012-03-24" },
+        { id: 63, merchant_id: 20, unit_price: 20_000, created_at: "2012-03-24", updated_at: "2012-03-24" },
+        { id: 65, merchant_id: 30, unit_price: 30_000, created_at: "2012-03-24", updated_at: "2012-03-24" },
+        { id: 66, merchant_id: 30, unit_price: 100_000, created_at: "2012-03-24", updated_at: "2012-03-24" }
+      ],
+
+      merchants: [
+          { id: 10, name: "merchant_1" },
+          { id: 20, name: "merchant_2" },
+          { id: 30, name: "merchant_3" }
+      ],
+
+      invoices: [
+        { id: 1, merchant_id: 10, created_at: "2012-03-24" },
+        { id: 2, merchant_id: 20, created_at: "2012-03-25" },
+        { id: 3, merchant_id: 30, created_at: "2012-03-24" },
+        { id: 4, merchant_id: 30, created_at: "2012-03-24" }
+      ],
+
+      invoice_items: [
+        { id: 100, invoice_id: 1, item_id: 61, quantity:  5,  unit_price: 10_000 },
+        { id: 200, invoice_id: 2, item_id: 63, quantity: 10,  unit_price: 20_000 },
+        { id: 300, invoice_id: 3, item_id: 65, quantity:  5,  unit_price: 30_000 },
+        { id: 400, invoice_id: 4, item_id: 66, quantity:  5,  unit_price: 100_000 }
+      ],
+
+      transactions: [
+        { id: 6, invoice_id: 1, result: "success" },
+        { id: 7, invoice_id: 2, result: "success" },
+        { id: 8, invoice_id: 3, result: "success" },
+        { id: 9, invoice_id: 4, result: "failed"  },
+      ]
+      }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.